modules/tinymce/src/plugins/legacyoutput/test/ts/browser/LegacyOutputPluginTest.ts

```diff
@@ -1,152 +1,176 @@
+  });
+
+  it('TBA: Justifycenter', () => {
+    const editor = hook.editor();
+    editor.setContent('<p>text</p>');
+    TinySelections.setSelection(editor, [ 0 ], 0, [ 0, 0 ], 4);
+
+    editor.execCommand('justifycenter');
+    TinyAssertions.assertContent(editor, '<p align="center">text</p>');
+  });
+
+  it('TBA: Justifyright', () => {
+    const editor = hook.editor();
+    editor.setContent('<p>text</p>');
+    TinySelections.setSelection(editor, [ 0 ], 0, [ 0, 0 ], 4);
+
+    editor.execCommand('justifyright');
+    TinyAssertions.assertContent(editor, '<p align="right">text</p>');
+  });
+
+  it('TBA: Justifyfull', () => {
+    const editor = hook.editor();
+    editor.setContent('<p>text</p>');
+    TinySelections.setSelection(editor, [ 0 ], 0, [ 0, 0 ], 4);
+
+    editor.execCommand('justifyfull');
+    TinyAssertions.assertContent(editor, '<p align="justify">text</p>');
+  });
+
+  it('TBA: Justifycenter image', () => {
+    const editor = hook.editor();
+    editor.setContent('<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAwAAAAMCAYAAABWdVznAAAAF0lEQVR42mP8/5/hPwMJgHFUw6gG7AAAXVgj6XowjMAAAAAASUVORK5CYII=" /></p>');
+    TinySelections.setSelection(editor, [ 0 ], 0, [ 0 ], 0);
+
+    editor.execCommand('justifycenter');
+    TinyAssertions.assertContent(editor, '<p align="center"><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAwAAAAMCAYAAABWdVznAAAAF0lEQVR42mP8/5/hPwMJgHFUw6gG7AAAXVgj6XowjMAAAAAASUVORK5CYII=" /></p>');
+  });
+
+  it('TBA: Remove text color', () => {
+    const editor = hook.editor();
+    const format: any = 'forecolor';
+    editor.setContent('<p><font color="red">text</font></p>');
+    TinySelections.setSelection(editor, [ 0, 0 ], 0, [ 0, 0, 0 ], 4);
+
+    editor.execCommand('mceRemoveTextcolor', format);
+    TinyAssertions.assertContent(editor, '<p>text</p>');
+  });
+
+  it('TBA: Remove background color', () => {
+    const editor = hook.editor();
+    const format: any = 'hilitecolor';
+    editor.setContent('<p><font style="background-color: red">text</font></p>');
+    TinySelections.setSelection(editor, [ 0, 0 ], 0, [ 0, 0, 0 ], 4);
+
+    editor.execCommand('mceRemoveTextcolor', format);
+    TinyAssertions.assertContent(editor, '<p>text</p>');
+  });
+
+  it('TINY-4741: Convert bold to span if styling attributes are present on format removal', () => {
+    const editor = hook.editor();
+    editor.setContent('<p><b class="abc" style="color: red; font-size: 20px;" data-test="2">text</b></p>');
+    TinySelections.setSelection(editor, [ 0, 0 ], 0, [ 0, 0, 0 ], 4);
+
+    editor.execCommand('bold');
+    TinyAssertions.assertContent(editor, '<p><span class="abc" style="color: red; font-size: 20px;">text</span></p>');
+  });
+
+  it('TBA: Formats registered before loading initial content', () => {
+    const formats = formatsCell.get();
+    assert.deepInclude(formats.bold[0], { inline: 'b', remove: 'all', deep: true, split: true, preserve_attributes: [ 'class', 'style' ] });
```

Comment:
Should these be `deepEqual` as that'd be equivalent to the previous version? as `deepInclude` allows other items or it to be at a different depth IIRC.